maya/core.py

```diff
@@ -417,6 +425,14 @@ def intersects(self, maya_interval):
     def midpoint(self):
         return self.start.add(seconds=(self.duration / 2))
 
+    @property
+    def aoe(self):
+        """
+        Return True if the interval has not passed anywhere on earth.
+        See also: http://www.ieee802.org/16/aoe.html
+        """
+        return self.start.aoe or self.end.aoe
```

Comment:
This isn't correct. You also need to check if the inverval is active, not just the endpoints. `self.start <= now < self.end`. (Adjust inclusivity as needed.)